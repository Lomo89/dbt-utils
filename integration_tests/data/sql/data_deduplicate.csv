user_id,event,version,category
1,play,a,green
1,play,b,NULL
1,play,NULL,green
2,pause,a,NULL
