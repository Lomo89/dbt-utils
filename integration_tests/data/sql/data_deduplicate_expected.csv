user_id,event,version,category
1,play,b,blue
